test/parallel/test-repl-null-thrown.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const repl = require('repl');
+const assert = require('assert');
+const Stream = require('stream');
+
+const output = new Stream();
+let text = '';
+output.write = output.pause = output.resume = function(buf) {
+  text += buf.toString();
+};
+
+const replserver = repl.start({
+  output: output,
+  input: process.stdin
+});
+
+replserver.emit('line', 'process.nextTick(() => { throw null; })');
```

Comment:
@TimothyGu yes, to reproduce the issue - without the nextTick it doesn't crash the REPL.